webui/src/pages/repositories/repository/compare.jsx

```diff
@@ -73,61 +89,82 @@ const CompareList = ({ repo, reference, compareReference, prefix, onSelectRef, o
+                    <>
+                        <ComingSoonModal display={showComingSoonModal}
+                                         onCancel={() => setShowComingSoonModal(false)}>
+                            <div>lakeFS Delta Lake tables diff is under development</div>
+                        </ComingSoonModal>
+                        <ExperimentalOverlayTooltip show={showDeltaDiffButton}>
+                            <Button className="action-bar"
+                                    variant="primary"
+                                    disabled={false}
+                                    onClick={() => {
+                                        setShowComingSoonModal(true);
+                                        sendDeltaDiffStats();
+                                    }}>
+                                <DiffIcon/> Compare Delta Lake tables
+                            </Button>
+                        </ExperimentalOverlayTooltip>
+                        <Card>
+                            <Card.Header>
+                                <span className="float-left">
+                                    {(delimiter !== "") && (
+                                        <URINavigator
+                                            path={prefix}
+                                            reference={reference}
+                                            relativeTo={relativeTitle(reference, compareReference)}
+                                            repo={repo}
+                                            pathURLBuilder={(params, query) => {
+                                                const q = {
+                                                    delimiter: "/",
+                                                    prefix: query.path,
+                                                };
+                                                if (compareReference)
+                                                    q.compare = compareReference.id;
+                                                if (reference)
+                                                    q.ref = reference.id;
+                                                return {
+                                                    pathname: '/repositories/:repoId/compare',
+                                                    params: {repoId: repo.id},
+                                                    query: q
+                                                };
+                                            }}/>
+                                    )}
+                                </span>
+                            </Card.Header>
+                            <Card.Body>
+                                <Table borderless size="sm">
+                                    <tbody>
+                                    {results.map(entry => {
+                                        if(!showDeltaDiffButton && entry.path.includes("_delta_log")) {
```

Comment:
This won't work if the delta table is in from a second order. 
See the example below - 
![Screen Shot 2023-01-10 at 10 25 29](https://user-images.githubusercontent.com/14786381/211499183-2460ca1c-03d4-4ad9-9564-13af55917f65.png)

I think that we should always show the button, and in the delta comparison view add the logic that only displays delta tables.

Reply:
done